reana/reana_dev/run.py

```diff
@@ -615,23 +615,31 @@ def _return_missing_output_files(component: str, workflow_name: str) -> List[str
             exit_status = 1
             exit_message = "RUNNING"
 
-        failed_report_message = ""
         if run_statistics["failed"]:
-            failed_report_message = f"({', '.join(run_statistics['failed'])})"
             exit_message = "FAILED"
             exit_status = 2
 
         submitted_workflows = sorted(
             [item for workflows in run_statistics.values() for item in workflows]
         )
 
+        def _format_report(workflows) -> str:
+            return f": {', '.join(workflows)}" if workflows else ""
```

Comment:
I did. Although, `_format_report` was only created to keep `:` or not depending on the list size. So not sure how much it will benefit in `_test_matrix`. But why not.